modules/contact/test/tests/mortar_dynamics/tests

```diff
@@ -165,7 +156,7 @@
     min_ad_size = 100
     ad_indexing_type = 'global'
     requirement = 'The system shall solve a dynamic 3D frictional one-element bouncing block problem '
-                  'with mortar constraint using nodal-attached geometry and a friction coefficient '
+                  'with mortar constraints using nodal geometry and a friction coefficient '
```

Comment:
I'm not asking you change anything; I'm just asking for my own edification: what is a nodal geometry? Haha I know I'm probably demonstrating some kind of wild ignorance here